### PR TITLE
Add secondary menu items as an alternative to using multiple icons

### PR DESCRIPTION
**Pull request by robxnano**
.desktop files have the ability to define multiple shortcuts with their own names and commands. If you add these to the existing shortcuts, there wouldn't be a need to have separate icons for GUI and non-GUI modes. Example of how they appear: 
Screenshot from 2022-09-07 23-36-27

Secondary menu items are supported on most major desktop environments, and are typically accessed by right-clicking on the launcher or taskbar icons.
I added an extra command-line argument to the AppRun wrapper to create a single shortcut. Due to the added complexity of the .desktop files, I had to use sed to insert the titles and paths, and I haven't had time to set up a Docker instance to test the build process yet. But please take a look and see how it could be useful.

### CODE REVIEW

File: docker/build/apprun-hook-install.sh [hunk 1]
```diff
@@ -11,17 +11,20 @@
...
 	fi
 	
 	# quick return if no recognized argument is present
-	if [ "${1:-}" != "--install" ] && [ "${1:-}" != "--uninstall" ] && [ "${1:-}" != "--add-menu-items" ] && [ "${1:-}" != "--remove-menu-items" ]
+	if [ "${1:-}" != "--install" ] && [ "${1:-}" != "--uninstall" ] && [ "${1:-}" != "--add-menu-item" ] && [ "${1:-}" != "--add-menu-items" ] && [ "${1:-}" != "--remove-menu-items" ]
 	then
 		return
 	fi
 	
+	APPIMAGE="/home/rob/Apps/SheepShaver/SheepShaver-x86_64.AppImage"
```
Comment (Korkman) [inline docker/build/apprun-hook-install.sh:27]: This needs cleaning

**Review by Korkman — CHANGES_REQUESTED** [review]
Great idea! I like it. This also eases integration with AppImageLauncher (having two desktop files breaks compatibility).

**Review by Korkman — APPROVED** [review]
Will test locally and clean-up now

**Comment by Korkman** [comment]
I modified your changes a bit. The result is now merged in main and available on the [release page](https://github.com/Korkman/macemu-appimage-builder/releases). Thank you for contributing!

**Comment by robxnano** [comment]
Thanks for fixing it up for me - it works great!

**Pull request merged**